docs/fsharp/style-guide/formatting.md

```diff
@@ -884,6 +884,33 @@ let makeStreamReader x = new System.IO.StreamReader(path=x)
 let makeStreamReader x = new System.IO.StreamReader(path = x)
 ```
 
+### Formatting constructors, static members and member invocations
+
+If the expression is long, use newlines and indent one scope, rather than indenting to the bracket.
+
+```fsharp
+let person =
+    new Person(
+        argument1,
+        argument2
+    )
+
+let p = new Person(a, b) // keep in one line if short
```

Comment:
This should be pulled into its own section, like so:

```
If the expression for calling a constructor is short, keep it on one line:

<example>
```

Then the bit about longer expressions can come afterwards.

This way we don't need the user to look at a code comment to see the guideline.

Reply:
Good point.